@ecomplus/storefront-app/src/components/html/EcOrderInfo.html

```diff
@@ -234,6 +234,18 @@
               {{ i19reopenOrder }}
             </button>
           </template>
+          <div class="order-info__buyAgain">
```

Comment:
```suggestion
          <div class="order-info__buy-again">
```

Sem maiúsculas em nomes de classes.